packages/connectors/connector-linkedin/src/constant.ts

```diff
@@ -0,0 +1,53 @@
+import type { ConnectorMetadata } from '@logto/connector-kit';
+import { ConnectorPlatform, ConnectorConfigFormItemType } from '@logto/connector-kit';
+
+export const authorizationEndpoint = 'https://www.linkedin.com/oauth/v2/authorization';
+export const defaultScope = 'openid profile email';
+export const accessTokenEndpoint = 'https://www.linkedin.com/oauth/v2/accessToken';
+export const userInfoEndpoint = 'https://api.linkedin.com/v2/userinfo';
```

Comment:
better add link to docs for future reference.

Reply:
updated